jenkins/compare.sh

```diff
@@ -68,6 +72,7 @@ else
 	URL_GENERATOR="$BUILD_URL/Generator_20Diff"
 fi
 
+printf "✅ [API Diff (from PR only)](%s)" "$URL_API" >> "$WORKSPACE/jenkins/pr-comments.md"
```

Comment:
This line (and the equivalent line just below in the same file) should not be included, since it would print the API Diff / Generator Diff twice.